### PR TITLE
Types @ v1.1

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What?
Bumps @guardian/type to v `1.1.0`

## What else?
Removes usage of optional chaining

## Why?
We want access to the `PhotoEssay` design type in DCR and so we need to update this package in each dependency

### CODE REVIEW

**Comment by oliverlloyd** [comment]
[@guardian/types v1.1 for atoms-rendering](https://trello.com/c/fkfU8sZB/2310-guardian-types-v11-for-atoms-rendering)

File: src/PersonalityQuiz.tsx [hunk 1]
```diff
@@ -214,7 +214,7 @@ export const PersonalityQuizAtom = ({
                 <Button
                     type="submit"
                     onClick={onSubmit}
-                    onKeyDown={(e) => {
+                    onKeyDown={(e: any) => {
```
Comment (liywjl) [inline src/PersonalityQuiz.tsx:217]:
```suggestion
                    onKeyDown={(e: React.KeyboardEvent<HTMLButtonElement>) => {
```

**Review by liywjl — APPROVED** [review]
Great work as always! 👍 I assume a lot of the challenge was figuring out how to get the build to work in storybook?

I am curious what `loader: require.resolve('ts-loader'),` does this try use `ts-loader` from from outside the repo?

File: src/PersonalityQuiz.tsx (hunk 1 above)
Comment (oliverlloyd) [reply src/PersonalityQuiz.tsx]: Thanks!

File: src/YoutubeAtom.tsx [hunk 2]
```diff
@@ -209,70 +209,81 @@ export const YoutubeAtom = ({
         let hasSent50Event = false;
         let hasSent75Event = false;
 
-        const listener = player.current?.on(
```
Comment (oliverlloyd) [inline src/YoutubeAtom.tsx:212]: @liywjl these diffs look wrong to me - can you check if I'm removing code that I shouldn't be here?

Comment (liywjl) [reply src/YoutubeAtom.tsx:212]: you are just removing optional chaining in this case. So its fine in my eyes

Comment (oliverlloyd) [reply src/YoutubeAtom.tsx:212]: But if you look at the larger diff, things seen very off, no?

Comment (oliverlloyd) [reply src/YoutubeAtom.tsx:212]: Double checked with @liywjl and these diffs are fine

**Pull request merged**